apt/private/deb_translate_lock.bzl

```diff
@@ -20,12 +22,87 @@ _DEB_IMPORT_TMPL = '''\
     )
 '''
 
-_BUILD_TMPL = """\
-exports_files(glob(['packages.bzl']))
+_PACKAGE_TEMPLATE = """\
+alias(
+    name = "{target_name}",
+    actual = select({data_targets}),
+    visibility = ["//visibility:public"],
+)
+
+alias(
+    name = "control",
+    actual = select({control_targets}),
+    visibility = ["//visibility:public"],
+)
+"""
+
+_ROOT_BUILD_TMPL = """\
+"Generated by rules_distroless. DO NOT EDIT."
+
+load("@rules_distroless//apt:defs.bzl", "dpkg_status")
+
+exports_files(['packages.bzl'])
+
+_ARCHITECTURE_MAP = {{
+    "amd64": "x86_64",
+    "arm64": "aarch64",
+    "ppc64el": "ppc",
+    "mips64el": "mips64"
+}}
+
+_ARCHITECTURES = {architectures}
+
+# See officially supported platforms at https://www.debian.org/releases/stable/armel/ch02s01.en.html
```

Comment:
From #100:
https://github.com/jjmaestro/rules_distroless/blob/b0f06d770cb6702179aa7fc994d1d1541cb4e8f7/apt/private/util.bzl#L3-L15

I added it in `util.bzl` instead of putting it inside the repo. Any advantages to doing this? I guess it's so you can put the `config_setting` in the `BUILD`. I still don't know enough to understand the advantages but it looks good here. I'd change the comment and the map, though. I think mine is a better fit.

Reply:
It's almost same except that its not in two places and easy to follow.